### PR TITLE
Import test cases from DVCS for Device Orientation Event

### PR DESCRIPTION
**Pull request by zqzhang**
https://dvcs.w3.org/hg/geo/rev/5ee2a7297634

### CODE REVIEW

File: orientation-event/free-fall-manual.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <head>
+    <title>DeviceMotionEvent: A device in free-fall, with the screen horizontal and upmost</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="author' title='Mosquito FP7">
+    <link rel="reviewer author" title="Zhiqiang Zhang" href="mailto:zhiqiang.zhang@intel.com"> <!-- 2013-10-14 -->
+  </head>
+  <body>
+    <p>Free fall the device to run the test, with the screen horizontal and upmost.</p>
+  	<div id="log"></div>
```
Comment (Honry) [inline orientation-event/free-fall-manual.html]: Indent not consistent.

Comment (Honry) [inline orientation-event/free-fall-manual.html]: Please fix all indent issue in other files.

File: orientation-event/horizontal-surface-manual.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      window.addEventListener("deviceorientation", function(e) {
+        if (!run) {
+          run = true;
+          gamma = e.gamma; // Gamma : angle par rapport a x
+          beta  = e.beta;  // Beta : angle par rapport a y
+          alpha = e.alpha; // Alpha : orientation (N-S-E-O)
+        }
+      }, false);
+
+      alert("Put the device on a horizontal surface.");
```
Comment (Honry) [inline orientation-event/horizontal-surface-manual.html]: Better to convert this hint to a text tag.

File: orientation-event/horizontal-surface-manual.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <div id="log"></div>
+    <script>
+      var gamma, beta, alpha;
+      var run = false;
+      window.addEventListener("deviceorientation", function(e) {
+        if (!run) {
+          run = true;
+          gamma = e.gamma; // Gamma : angle par rapport a x
+          beta  = e.beta;  // Beta : angle par rapport a y
+          alpha = e.alpha; // Alpha : orientation (N-S-E-O)
```
Comment (Honry) [inline orientation-event/horizontal-surface-manual.html]: ```alpha``` in this test is unused. Need a assertion for it.

**Comment by zqzhang** [comment]
@Honry thanks for the review. Your comments shall be addressed. Please take another look, thanks.

**Comment by Honry** [comment]
LGTM

**Pull request merged**